addons/account/views/account_payment_method.xml

```diff
@@ -12,4 +12,20 @@
         </field>
     </record>
 
+    <record id="view_account_payment_method_line_kanban_mobile" model="ir.ui.view">
+        <field name="name">account.payment.method.line.kanban</field>
+        <field name="model">account.payment.method.line</field>
+        <field name="arch" type="xml">
+            <kanban class="o_kanban_mobile">
+                <t t-name="kanban-box">
+                    <div t-attf-class="oe_kanban_card">
+                        <span>
+                            <field name="name" string="Payment Method Name"/> (<field name="journal_id"/>)
+                        </span>
```

Comment:
This would not work with translation
Only "Payment Method Name" would be translatable. RTL languages would also be broken :thinking: